tsconfig.json

```diff
@@ -1,12 +1,15 @@
 {
-    "extends": "@docusaurus/tsconfig/tsconfig.json",
+    "extends": "@docusaurus/tsconfig",
     "include": [
         "src/"
     ],
     "compilerOptions": {
       "baseUrl": ".",
       "downlevelIteration": true,
-      "moduleResolution": "Node16",
-      "module": "Node16"
+      "types": [
+        "node",
+        "@docusaurus/module-type-aliases",
+        "@docusaurus/theme-classic"
+      ],
```

Comment:
You probably don't need to restore this

If you import the preset types in your config file (making your TS config more typesafe), then the required ambient types gets registered as well.

See also: https://github.com/facebook/docusaurus/issues/10733#issuecomment-2520939228

Reply:
Thank you!